adagrams/game.py

```diff
@@ -1,13 +1,162 @@
-from random import randint
+from random import randint 
+
+def create_letter_pool():
+    letter_pool = []
+    letter_pool.extend(["A"] * 9)
+    letter_pool.extend(["B"] * 2)
+    letter_pool.extend(["C"] * 2)
+    letter_pool.extend(["D"] * 4)
+    letter_pool.extend(["E"] * 12)
+    letter_pool.extend(["F"] * 2)
+    letter_pool.extend(["G"] * 3)
+    letter_pool.extend(["H"] * 2)
+    letter_pool.extend(["I"] * 9)
+    letter_pool.extend(["J"] * 1)
+    letter_pool.extend(["K"] * 1)
+    letter_pool.extend(["L"] * 4)
+    letter_pool.extend(["M"] * 2)
+    letter_pool.extend(["N"] * 6)
+    letter_pool.extend(["O"] * 8)
+    letter_pool.extend(["P"] * 2)
+    letter_pool.extend(["Q"] * 1)
+    letter_pool.extend(["R"] * 6)
+    letter_pool.extend(["S"] * 4)
+    letter_pool.extend(["T"] * 6)
+    letter_pool.extend(["U"] * 4)
+    letter_pool.extend(["V"] * 2)
+    letter_pool.extend(["W"] * 2)
+    letter_pool.extend(["X"] * 1)
+    letter_pool.extend(["Y"] * 2)
+    letter_pool.extend(["Z"] * 1)
+    return letter_pool
```

Comment:
I love that you used a helper function here! Makes you code easier to debug in case of bugs, however it is not as DRY as it could be. We could make this more DRY by using a loop (recall that loops are perfect for when we want to use preform the same logic a number of times. It could look something like this: 
```python
def create_letter_pool():
  LETTERS_POOL = {
      "A":9,"B":2,"C":2,"D":4,
      "E":12,"F":2,"G":3,"H":2,
      "I":9,"J":1,"K":1,"L":4,"M":2,
      "N":6,"O":8,"P":2,"Q":1,
      "R":6,"S":4,"T":6,"U":4,
      "V":2,"W":2,"X":1,
      "Y":2,"Z":1
  }
  letter_list = []

  for letter, count in LETTER_POOL.items():
    letter_list.extend(list(letter) * count)
  
  return letter_list
```